utils/swift_build_support/swift_build_support/products/product_builder.py

```diff
@@ -0,0 +1,86 @@
+# swift_build_support/product_builders/product_builder.py -------*- python -*-
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ----------------------------------------------------------------------------
+
+import abc
+
+
+class ProductBuilder(object):
```

Comment:
Not a blocker in any way, but I'm curious why this class was put in a separate module instead of grouping it in the `product` module. I'm OK with either, but since it's so closely tied it seems maybe fitting for it to just live in the same module.

Reply:
Probably because I like the idea of one file, one type; but in Python that idea might not be followed and might not be the best anyway. I will see to move it into `product.py` instead.